app/src/main/java/ch/epfl/polybazaar/notifications/NotificationService.java

```diff
@@ -19,14 +26,24 @@
 import ch.epfl.polybazaar.user.User;
 
 public class NotificationService extends FirebaseMessagingService {
+
+    LocalBroadcastManager broadcastManager;
     public NotificationService() {
     }
```

Comment:
Is the constructor defined specifically to prevent from calling `this.super()`? This would be weird, if it is not the case an empty constructor is not necessary since a default constructor will be added.

Reply:
I did not realize that this code was from another PR, but it would be good to clarify this since it is confusing anyway (if it is intended, just write a comment with an explanation).

Reply:
Yes the constructor is useless now I removed it